New governance proposal

Calvina24 opened the pull request:
You will find here a proposal of a new governance for the iTowns project.
In order to discuss about these changes, please read the Issue : https://github.com/iTowns/itowns-governance/issues/2 
And its corresponding sub-issues.

Calvina24 committed:
Add new governance proposal files

Calvina24 committed:
Delete membership.md

Calvina24 committed:
Delete proposal.md

Calvina24 committed:
Delete README.md

Calvina24 committed:
Update and rename [proposal] README.txt to README.txt

Calvina24 committed:
Update and rename [proposal] membership.txt to membership.txt

Calvina24 committed:
Update and rename [proposal] proposal.txt to proposal.txt

Calvina24 committed:
Update and rename README.txt to README.md

Calvina24 committed:
Update and rename membership.txt to membership.md

Calvina24 committed:
Update and rename proposal.txt to proposal.md

Calvina24 committed:
hackathon - Revise iTowns governance document for clarity and updates

Calvina24 committed:
Revise iTowns governance structure and documentation

Updated the governance document to clarify roles, processes, and values.

Calvina24 committed:
Revise governance document in README.md

Updated the README.md to reflect governance structure and values.

TODO : spécifier comment se passe le vote des core contributors.
quel type de majorité est nécessaire par type de décision

Calvina24 committed:
Revise removal vote README.md

Updated the README.md . à valider avec l'équipe

Calvina24 committed:
Revise membership.md for clarity and formatting

Updated membership document to clarify PSC members and core contributors.

Calvina24 committed:
Revise proposal and contribution process documentation

changement : double majority to absolute majority
changement : double majority to majority

Calvina24 committed:
remaining TODO 

vote type
links

Calvina24 committed:
discord channel

jailln commented on membership.md:
```diff
@@ -1,30 +1,21 @@
+- JAILLOT Vincent(Ciril Group representative)
+- LAMEGER Caroline (Ciril Group representative)
+
+## Core Contributors
```
D'après nos derniers échanges la liste serait plutôt:

- BOUILLAGUET Quentin
- DUBOURG Irenee
- DUHEN Raphaël
- ETOURNEAU Kevin
- GERMERIE-GUIZOUARN Madec
- GULLIENT Anthony
- JAILLOT Vincent
- RIPPLINGER Victor
- TOROMANOFF François

jailln commented on proposal.md:
```diff
@@ -1,85 +1,86 @@
+- **Improvement** of an existing feature or component
+- **Addition** of a new feature.
+  
+*Note: Proposals are only required for improvements or additions of features. Bug reports are sufficient for proposing a fix.*
```
I think we could remove the "types of contributions" part.

jailln commented on proposal.md:
```diff
@@ -1,85 +1,86 @@
+  
+*Note: Proposals are only required for improvements or additions of features. Bug reports are sufficient for proposing a fix.*
+
+### Bug Discovery and Correction (proposal not required)
```
remove "(proposal not required)", i think readers won't understand what it means here

Calvina24 replied:
I will then remove the section to avoid misunderstandings (the bug report process is explained in the CONTRIBUTING.md)

jailln commented on proposal.md:
```diff
@@ -1,85 +1,86 @@
+2. A contributor submits a fix by opening a pull request linked to the corresponding issue in the description
+
+
+### Significant improvements and new features (proposal required)
```
remove "(proposal required)". It is explained in the text anyway

jailln commented on proposal.md:
```diff
@@ -1,85 +1,86 @@
+  
+If at least one of the Core Contributors expresses a non-favorable advice to the proposal, the issue can remain open, the proposal can be amended and discussions can continue until the negative advice is removed. Meanwhile, no implementation corresponding to this proposal should be merged.
+
+If a consensus is reached, or no negative advice from Core Contributors or PSC has been expressed 1 month after the last comment on the proposal, it is considered as acceptable.
```
We could replace all that by: A proposal is considered accepted if at least one core contributor validates it or if no negative response from Core Contributors has been given 1 month after the last comment on the proposal.

jailln commented on proposal.md:
```diff
@@ -1,85 +1,86 @@
+
+Only core contributors are considered as valid reviewers.
+
+PSC members are also expected to assess the functional aspect of the implementation.
```
Remove this phrase

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+
+## Preamble
+
+**iTowns** is a **geographic commons**, developed collectively by a diverse community of contributors, comprising independent developers, public organizations, research laboratories, and private companies.
```
remove "is a **geographic commons**"

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+
+**iTowns** is a **geographic commons**, developed collectively by a diverse community of contributors, comprising independent developers, public organizations, research laboratories, and private companies.
+
+The commons is governed by **rules** that guarantee its expansion and the immutability of its open, shared nature. This document formalizes the **core values**, **community structure**, and **current governance** of the [`iTowns`]([itowns](https://github.com/iTowns/itowns) project.
```
"The commons" => iTowns

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+
+If you wish to propose changes on different topics, please create a separate PR or issue for each topic.
+ 
+Contact us at [iTowns@ign.fr](mailto:iTowns@ign.fr).
```
Why this line here? => To remove, already at the end of the file.

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+
+
+### Becoming a core contributor
+Achieving status of Core Contributor requires significant contributions to the project, a demonstrated ability to produce high-quality code or other value to the project, and adherence to the guidelines outlined in the code of conduct (TODO link), [CODING.md](https://github.com/iTowns/itowns/blob/master/CODING.md) and [CONTRIBUTING.md](https://github.com/iTowns/itowns/blob/master/CONTRIBUTING.md) ...
```
don't forget the TODO Link to the code of conduct and can you please add the code of conduct we sent you please ?

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+
+This role is granted on an individual basis (not as part of an organization). 
+
+It may be rescinded for rule violations or if contributions cease for a 6 months period. Revocation requires a absolute majority vote of existing core contributors, excluding the concerned individual.
```
To replace by:

A role may be revoked due to rule violations or inactivity (6+ months without contributions). This requires an absolute majority vote from active core contributors, excluding the person concerned. Revocation is automatic after 12 consecutive months without contributions.

jailln commented on proposal.md:
```diff
@@ -1,85 +1,86 @@
+# Proposal, Contribution and Review Process
+
+## Introduction
+Before making any contributions, it is important for contributors and the community at large to open discussions ( proposals ) about addition or significant improvement of features.
```
Update to: "Before making any significant improvement, contributors are required to open discussions ( proposals).

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+This role is granted on an individual basis (not as part of an organization). 
+
+It may be rescinded for rule violations or if contributions cease for a 6 months period. Revocation requires a absolute majority vote of existing core contributors, excluding the concerned individual.
+Should this person become active again, they will certainly be welcome, but would require a nomination. Revocation is automatic if contributions cease for a one year period.
```
remove "Revocation is automatic if contributions cease for a one year period."

jailln commented on README.md:
```diff
@@ -1,164 +1,145 @@
+- A new PSC member can be invited by the current PSC members. The nominated member may be an individual or an organization. In the latter case, a representative will be appointed by the organization.
+- A vote will follow. Only existing PSC committee members may vote on new members. Nominees must receive an absolute majority vote from existing members to be added to the PSC.
+
+A member may be removed from the committee if their core contributors remain inactive for a period of six months, or upon a proposal submitted to the committee and subject to an absolute majority vote by the other members of the PSC. Removal proposal must be motivated and its rationale explained in detail, with references to the project's values and global strategy and purpose.
```
"their core contributors remain inactive for a period of six months" => "They haven't had any core contributors for the past 6 months"

Calvina24 committed:
Update membership.md with actual core contributors

Calvina24 committed:
Revise proposal and contribution process documentation

Modifications according to @jailln comments

Calvina24 committed:
Add Code of Conduct for iTowns project

This document outlines the Code of Conduct for the iTowns project, including expected behaviors, unacceptable behaviors, and reporting procedures.

Calvina24 committed:
Delete CODE_OF_CONDUCT.md

wrong repository. My bad

Calvina24 committed:
Revise governance documentation in README.md

According to @jailln comments

Calvina24 committed:
Merge branch 'master' into New-governance-proposal
